src/OrbitBase/FileTest.cpp

```diff
@@ -228,4 +228,42 @@ TEST(File, ReadFullyAtOffsetSmoke) {
   EXPECT_STREQ(buf.data(), "");
 }
 
+TEST(File, ReadStructureAtOffset) {
+  struct TestStructure {
+    uint64_t u64;
+    std::array<char, 8> char_array;
+  };
+
+  auto tmp_file_or_error = TemporaryFile::Create();
+  ASSERT_TRUE(tmp_file_or_error.has_value()) << tmp_file_or_error.error().message();
+
+  auto fd_or_error = OpenExistingFileForReadWrite(tmp_file_or_error.value().file_path());
+  ASSERT_TRUE(fd_or_error.has_value()) << fd_or_error.error().message();
+
+  auto fd = std::move(fd_or_error.value());
+
+  const uint64_t u64_value = 1121;
+  const char* char_array = "abcdefg";
```

Comment:
I'd make them `constexpr` and name them like `kConstant`s.

Reply:
done